Berlin/BerOrOct59.xml

```diff
@@ -26,43 +30,250 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
+                         <locus from="86v" to="97r"></locus>
+                         <title type="complete" ref="LIT1113Anqasa"></title>
+                         <textLang mainLang="gez"> </textLang>
+                         <incipit xml:lang="gez"><hi rend="rubric"> በስመ፡ አብ፡ ወወልድ ወመንፈስ፡ ቅዱስ።</hi> አሐዱ፡ አምላክ። ጸሎት፡<hi rend="rubric">
+                         ዘእግዚእትነ፡ ማርያም። </hi> ዘጸለየት፡ ወትቤ። በርታፍ<hi rend="rubric">ዮስ። ተሐዝልፍዮስ። ዐምድ፡</hi> ወዮስ። ዘቆሮልዮስ።
+                         </incipit>
+                         </msItem>
+                  <msItem  xml:id="p2_i7">
+                         <locus from="98r" to="108r"></locus>
+                         <title type="complete" ref="LIT2509Weddas"></title>
+                         <textLang mainLang="gez"> </textLang>
+                         <incipit xml:lang="gez"> <hi rend="rubric">አውሎግሶን፡ ኢየሱስ፡ ክርስቶስ፡ ለይኩን፡ ብርሃኑ" ለእግዚአብ</hi>ሔር፡ አምላከነ፡ ላዕሌነ፡ ወይሰርሕ፡
+                           ለነ፡ ተግባረ፡ እደዊነ። ወእም<hi rend="rubric">ዝ፡ ንጽሐፍ፡ ወንዘከር ውዳሴሃ፡ ለእግዚእትነ፡ ቅድስት፡ ማርያም፡</hi> ድንግል።
+                         </incipit>
+                         </msItem>
+                 <msItem  xml:id="p2_i8">
+                             <locus from="109r" to="142v"></locus>
+                             <title> Prayer of supplication, pleading and praise to Mary. </title>
+                             <textLang mainLang="gez"> </textLang>
+                             <incipit xml:lang="gez"> <hi rend="rubric">ጸሎት፡ ወ</hi>ስእለት፡ ወአስተብቊዖት፡ ወውዳስያት፡ ወተማኅፅኖት፡ ኀበ፡ እግዝእትነ፡ ማርያም፡ ድንግል፡
+                               </incipit>
+                           </msItem>
+                <msItem  xml:id="p2_i9">
+                              <locus from="145r" to="161r"></locus>
+                              <title ref="LIT1295Dersan#HomilyHedar">Dǝrsāna Mikāʾel Homily by Timothy (Damātēwos), Patriarch of Alexandria for the 12th of Ḫǝdār.</title>
+                              <textLang mainLang="gez"> </textLang>
+                              <incipit xml:lang="gez"> <hi rend="rubric">በስመ፡ አብ፡ ወወልድ ወመን</hi>ፈስ፡ ቅዱስ። አሐዱ፡ አምላ<hi rend="rubric">ክ። ድርሳን፡ ነገር፡ ዘቅዱስ፡ አ</hi>ባ፡ ደማቴዎስ፡
+                              ሊቀ፡ ጳጳሳት፡ <hi rend="rubric">ለእስከንድርያ፡ በበዓለ፡ ሊቀ፡</hi> መላእክት፡ ሚካኤል፡
+                              </incipit>
+                        </msItem>
                </msContents>
+
+        <physDesc>
+
+             <objectDesc form="Codex">
+                <supportDesc>
+                   <support>
+                      <material key="parchment"/>
+                   </support>
+                 <extent>
+                   <measure unit="leaf">i+167</measure>
+                    <dimensions type="outer" unit="mm">
+                       <height>150</height>
+                       <width>110</width>
+                       <depth>55</depth>
+                    </dimensions>
+                  </extent>
+
+                  <foliation>
+                     <list>
+                        <item resp="p2">
```

Comment:
Here, two systems of foliation/pagination are described. You can either leave the sentence as it is in a simple `<foliation>` element without  list and items, or keep the list and create one item per foliation system (note the xml:id - "resp" here is probably a mistake for "corresp", in which case a # would be needed before p2.)
```
<foliation>
<item xml:id="fol1">
Foliation in pencil in the lower right corner.
</item>
<item xml:id="fol2">
Pagination in pencil in the upper right corner.
</item>
</foliation>
```
(deleting the list and keeping it in one element is completely fine, this is just to show to you how the list is meant to be used)